src/uTensor/ops/Convolution_kernels.hpp

```diff
@@ -7,7 +7,7 @@
 
 namespace uTensor {
 
-enum Padding : uint8_t { VALID = 1, SAME = 2 };
+enum Padding : uint8_t { UNKNOWN = 0, VALID = 1, SAME = 2 };
```

Comment:
@mbartling 
The qDWS conv we have was taken from TFLM. But, this enum seems to be a generalization to the padding types, a fairly reasonable generalization indeed. However, we will have to think about where to draw the line in terms of cross-op-spec generalization and abstraction.